examples/descriptorbuffer/descriptorbuffer.cpp

```diff
@@ -204,8 +204,8 @@ class VulkanExample : public VulkanExampleBase
 		vkGetDescriptorSetLayoutBindingOffsetEXT(device, combinedImageDescriptor.setLayout, 0, &combinedImageDescriptor.layoutOffset);
 
 		// In order to copy resource descriptors to the correct place, we need to calculate aligned sizes
-		uniformDescriptor.layoutSize = vks::tools::alignedSize(uniformDescriptor.layoutSize, descriptorBufferProperties.descriptorBufferOffsetAlignment);
-		combinedImageDescriptor.layoutSize = vks::tools::alignedSize(combinedImageDescriptor.layoutSize, descriptorBufferProperties.descriptorBufferOffsetAlignment);
+		uniformDescriptor.layoutSize = vks::tools::alignedSize(static_cast<uint32_t>(uniformDescriptor.layoutSize), static_cast<uint32_t>(descriptorBufferProperties.descriptorBufferOffsetAlignment));
+		combinedImageDescriptor.layoutSize = vks::tools::alignedSize(static_cast<uint32_t>(combinedImageDescriptor.layoutSize), static_cast<uint32_t>(descriptorBufferProperties.descriptorBufferOffsetAlignment));
```

Comment:
For consideration: Maybe an additional prototype

    VkDeviceSize vks::tools::alignedSize(VkDeviceSize, VkDeviceSize);

would be useful, since all the involved types here are `VkDeviceSize`. In that case the casting would not be necessary in this case.